metricflow/model/objects/metric.py

```diff
@@ -27,10 +27,15 @@ class MetricType(ExtendedEnum):
 
 
 class MetricInputMeasure(PydanticCustomInputParser, HashableBaseModel):
-    """Provides a pointer to a measure along with metric-specific processing directives"""
+    """Provides a pointer to a measure along with metric-specific processing directives
+
+    If an alias is set, this will be used as the string name reference for this measure after the aggregation
+    phase in the SQL plan.
+    """
 
     name: str
-    # TODO add constraint handling property
+    constraint: Optional[WhereClauseConstraint]
+    alias: Optional[str]
```

Comment:
Would there be an issue if to two `MetricInputMeasure`s have the same alias? If so, we need to either add a new validation rule or update the unique naming validation rule to account for this. In the same vain, I assume there'd be an issue if an alias conflicts with an existing measure name?

Reply:
Yes. There are a bunch of subtleties with the alias attribute. I'm planning to add those validations in a separate PR.

Reply:
I'm a dumb dumb, you put that at the end of your PR description 🤦 my bad